### PR TITLE
Update pom.xml

### PR DESCRIPTION
**Pull request by oterrien**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.ote.test</groupId>
     <artifactId>ote.kata</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (oterrien) [inline pom.xml]: why

Comment (oterrienGmail) [inline pom.xml]: Ok I understand

Comment (oterrienGmail) [inline pom.xml]: Ok I understand

Comment (oterrienGmail) [inline pom.xml]: because

**Pull request closed without merging**